gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -992,8 +992,8 @@
      * Adds a {@link Vertex} with a default vertex label.
      *
      * @return the traversal with the {@link AddVertexStep} added
-     * @since 3.1.0-incubating
      * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#addvertex-step" target="_blank">Reference Documentation - AddVertex Step</a>
+     * @since 3.1.0-incubating
```

Comment:
a few weeks ago i added all the annotations and references, but it looks like your IDE didn't like my formatting so it just shuffled stuff around.